test/tests.py

```diff
@@ -160,3 +162,20 @@ def test_003_jupyterAwake(self):
         # Making sure that the Jupyter notebook is up and can be reached.
         response = requests.get(self.jupyter_endpoint)
         self.assertEqual(response.status_code, HTTPStatus.OK)
+
+    def test_004_negativeAuth(self):
```

Comment:
I would add what the test is doing to the function name. For example,
test_004_negativeAuth_InvalidCreds.

Reply:
Also I would add at least one more 0 to your test case numbers.